janus_core/md.py

```diff
@@ -360,20 +360,23 @@ def _optimize_structure(self) -> None:
                 self.logger.info("Minimizing at step %s", self.dyn.nsteps)
             optimize(self.struct, **self.minimize_kwargs)
 
-    def configure_filenames(self) -> None:
+    def configure_filenames(self, pressure: Optional[float] = None) -> None:
         """Setup filenames for output files."""
+        temp = ""
         if not self.file_prefix:
-            self.file_prefix = f"{self.struct_name}-{self.ensemble}-T{self.temp}"
+            self.file_prefix = f"{self.struct_name}-{self.ensemble}"
+            if pressure is not None:
+                 self.file_prefix = f"{self.file_prefix}-p{pressure}" 
+            if self.temp_start and self.temp_end:
+               temp = f"-T{self.temp_start}-T{self.temp_end}"
```

Comment:
I think I might have had this conversation already with @alinelena, but if the MD temperature is different to the final temperature, the MD temperature won't appear in the file name. This is probably fine, but just checking this is intentional.

I was initially going to suggest we might want to include the MD temperature if it differs from the final temperature, but actually I wonder if it makes sense to include anyway, to distinguish between simulations with and without MD at the end of heating?

Reply:
we fixed that one commits to show...

Reply:
Sorry, I'm not sure what you mean?

Also, sorry @harveydevereux I just saw that this was essentially your question at the start of the PR. My feeling is yes, ideally we'll have some indication that MD is run. Adding the extra temperature is probably the simplest option, but maybe there's a nicer format given that usually it should match the heating final temperature...

Reply:
I think @alinelena means I have that on my to-do to add the extra T if it is different to the temperature at the end of the heating. We talked about it at lunch.

Reply:
still ongoing with this, the MD T is added but currently always (not just if different)